src/schedule/zephyr_dp_schedule.c

```diff
@@ -403,7 +403,7 @@ int scheduler_dp_init(void)
 	scheduler_init(SOF_SCHEDULE_DP, &schedule_dp_ops, dp_sch);
 
 	/* init src of DP tick */
-	ret = schedule_task_init_ll(&dp_sch->task,
+	ret = schedule_task_init_ll(&dp_sch->ll_tick_src,
```

Comment:
Not related to this PR, but revisiting this code, one possibility would be to just use k_timer to run scheduler_dp_ll_tick() every 1ms. But, but , this would be from IRQ context, so reusing the LL thread should work as well.

Reply:
We need to run it when LL finishes, using of LL task is intentional